auction-app/src/main/java/com/auctionapp/model/product/Product.java

```diff
@@ -31,16 +32,9 @@ public class Product {
     @Column(name = "category")
     private EProductCategory category;
 
-    @JoinColumn(name = "user_id")
-    private Long userId;
-
-    public Product(String name, double starting_price, String description, Long userId, EProductCategory category) {
-        this.name = name;
-        this.starting_price = starting_price;
-        this.description = description;
-        this.userId = userId;
-        this.category = category;
-    }
+    @ManyToOne
+    @JoinColumn(name="user_id", referencedColumnName = "id")
+    private User user;
 
```

Comment:
Why we don't have a constructor here ?

Reply:
We have @NoArgsConstructor annotation for empty object, as i understand, previous product constructor without id was used in controller to save, update and etc. As we have now ProductDTO, it represents all required data. Actually i removed constructor, as intellj showed it in grey as never used, probably even didn;t noticed that. So not sure, i can restore as it was, maybe it would be used for other entities, but as we need userId, not whole User object, probably ProductDTO is enough

Reply:
Ok I see, thanks for explaination ! :D if it works without, let's keep it this way ;)